src/components/views/__tests__/CinemaHall.spec.js

```diff
@@ -0,0 +1,54 @@
+import { describe, it, expect } from "vitest";
+import { shallowMount } from "@vue/test-utils";
+import CinemaHall from "@/components/views/CinemaHall.vue";
+import rowLetter from "@/components/views/CinemaHall.vue";
+import isSeatTaken from "@/components/views/CinemaHall.vue"
+
+
+const createComponent = () =>
+  shallowMount(CinemaHall, {
+    props: { rows, seats, seatsTaken },
+  });
+
+const rows = 10;
+const seats = 10;
+const seatsTaken = ['A1'];
+
+describe("CinemaHall.vue", () => {
+  it("expects to be mounted", () => {
+    const wrapper = createComponent();
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("should display a given seats number in a row", () => {
+    const wrapper = createComponent();
+    expect(wrapper.findAll('.cinema-hall__row').length).toBe(seats)
+  });
```

Comment:
the test description is different than what test is checking 🤔  (`findAll` gets all elements matching the selector)